modules/OFConnectionManager2/module/src/ofconnectionmanager.c

```diff
@@ -420,6 +425,9 @@ listen_socket_ready(int socket_id, void *cookie, int read_ready,
         /* FIXME use AIM datatype when available */
         AIM_LOG_VERBOSE("Accepted cxn from [IPv6addr]:%d",
                         ntohs(sa6->sin6_port));
+    } else if (cxn_addr.ss_family == AF_UNIX) {
+        struct sockaddr_un *sun = (struct sockaddr_un*) &cxn_addr;
+        AIM_LOG_VERBOSE("Accepted cxn from %s", sun->sun_path);
```

Comment:
I don't think unix domain sockets have a useful peer address. When I tested it this log printed random binary junk.